dev-vcs/cvs/files/cvs-1.12.12-fix-implicit-function-declarations.patch

```diff
@@ -0,0 +1,31 @@
+Clang16 will not allow implicit function declarations by default,
+therefore we need to declare the correct functions from stdlib.h.
+
+Bug: https://bugs.gentoo.org/870835
+Bug: https://bugs.gentoo.org/882225
+
+Pascal Jäger <pascal.jaeger@leimstift.de> (2022-12-04)
+
+--- a/lib/sighandle.c
++++ b/lib/sighandle.c
+@@ -41,8 +41,8 @@
+ #include <stdlib.h>
+ #else
+ #if __STDC__
+-char *calloc(unsigned nelem, unsigned size);
+-char *malloc(unsigned size);
++void *calloc(size_t nelem, size_t size);
++void *malloc(size_t size);
+ #else
```

Comment:
why tho? I know we not gonna end up in *not _STD_C* but why should I patch it out entirely? It not hurting our build, is it?

Reply:
because it's asking for gratuitous conflicts down the road. Some configure check misfires, picks this code, and then you have different prototypes from `stdlib.h` conflciting with this one